datahub-web-react/src/app/permissions/policy/PolicyPrivilegeForm.tsx

```diff
@@ -321,6 +354,23 @@ export default function PolicyPrivilegeForm({
             : displayStr;
     };
 
+    const getResourceText = (policyMatch) => {
+        if (policyMatch === PolicyMatchCondition.Equals) {
+            return (
+                <Typography.Paragraph>
+                    Search for specific resources the policy should apply to. If <b>none</b> is selected, policy is
+                    applied to <b>all</b> resources of the given type.
+                </Typography.Paragraph>
+            );
+        }
+        return (
+            <Typography.Paragraph>
```

Comment:
This is introducing a non-trivial amount of new risk into this system by adding a lot of new complexity. 

Is there consideration about priority of exclusion inclusion rules and conflicting rules?